fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/Constants.java

```diff
@@ -0,0 +1,16 @@
+package com.github.dedis.popstellar.utility;
+
+public class Constants {
```

Comment:
Pure nitpicking, but why creating a whole new class ?
You could have defined it in the fragment classes.

Reply:
Because they are extras passed across activities, I thought it was cleaner that way. Plus I thought it might be useful to add some other constants in the future